### ISSUE TITLE
{Linked:Bug986547:}[Adaptive Cards > Designer] - "Interview feedback" text visually appears as heading but is not defined as heading programmatically.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9169]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 27766.1000)
Browser Version: Version 132.0.2186.2 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Navigate the screen in scan mode using 'H' key and observe the issue.

**Actual Result:**
"Interview feedback" text visually appears as heading but is not defined as heading programmatically.

**Expected Result:**
"Interview feedback" text should be programmatically defined as heading and it should be defined as <h2>.

**User Impact:**
Visually impaired users who depend on screen readers will get impacted if visual heading structure is not defined.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/3d0f0e41-3c1b-463f-a37b-57171304612b

![Image](https://github.com/user-attachments/assets/648490da-0168-40dd-bb6e-e1609528c98b)

![Image](https://github.com/user-attachments/assets/dc01d8a8-bedb-4edd-86d5-369ab9862095)

### Card JSON

```JSON
{
        "type": "AdaptiveCard",
        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
        "version": "1.4",
        "body": [
            {
                "type": "TextBlock",
                "text": "**Interview feedback**",
                "wrap": true
            },
            {
                "type": "TextBlock",
                "text": "Thank you for completing your interview round today. Please provide your feedback comments and recommendation for the following candidate based on your conversation with them.",
                "wrap": true
            }
        ]
}
```

### Sample Code

Originally tracking with https://msdyneng.visualstudio.com/FinOps/_workitems/edit/986547

**Comment by dony-james** [comment]
Please update the details about the bug fix here

**Comment by avanish2821** [comment]
@SatwikKrSharma I tried the above card but it didn't work

![Image](https://github.com/user-attachments/assets/7e6ba697-beba-4649-b282-cb42c36bc19f)

![Image](https://github.com/user-attachments/assets/2652a349-7fcf-42b8-b78c-126dc00d89fa)
